app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    hidden = False
+    alive = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        Animal.alive.append(self)
+
+    def __len__(self) -> int:
+        return len(Animal.alive)
+
+    def __repr__(self) -> str:
+        return (
+            f"{{Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}}}"
+        )
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = not self.hidden
```

Comment:
The `hidden` attribute should be an instance variable, but it is currently a class variable. You should initialize `self.hidden` in the `__init__` method of the `Animal` class.